project_timelog/res_config.py

```diff
@@ -1,27 +1,17 @@
 # -*- coding: utf-8 -*-
-from openerp.osv import fields, osv
-from openerp.http import request
+from openerp import models, fields
 
-class timelog_config_settings(osv.osv_memory):
+
+class TimelogConfigSettings(models.Model):
```

Comment:
You don't need real Model and also you get error on runbot

    class TimelogConfigSettings(models.Model):